bounties/level-1/create-bounties.md

```diff
@@ -0,0 +1,51 @@
+# Creating Bounty Suggestions
+
+## Award
+| Hunter | Agent | Type
+| :- | :- | :-
+| 200 XLM per submission | 0 Credits | Continuous
+
+## Context
+https://github.com/tyvdh/stellar-quest-bounties/issues?q=create-bounties.md
```

Comment:
Let's exclude the .md suffix just in the chance folks don't use the .md extension.